client-app/package.json

```diff
@@ -12,9 +12,11 @@
     "lint:js": "eslint --ext .js,.jsx,.ts,.tsx .",
     "lint:styles": "stylelint \"**/*.s?(a|c)ss\"",
     "start": "yarn install && cross-env NODE_OPTIONS=--max_old_space_size=3072 webpack-dev-server",
+    "start2": "yarn start --env devGrailsPort=8082 devWebpackPort=3002",
```

Comment:
What are these updates doing? Does this version needs it own port- meaning will 8080 also be in use when running in this manner?

Reply:
yes, indeed, this is for running a second client in development that will hit the second server

Reply:
Thanks for the update. Perhaps a name update to something like `startSecondClient` / `startSecondClientWithHoist`?

Reply:
Agree the name 2 sounds like a potential replacement!  Great catch.  But I suggest just changing out the '2' with `instance2` e.g. `startInstance2`, `startInstance2WithHoist